src/diffusers/pipelines/deprecated/versatile_diffusion/modeling_text_unet.py

```diff
@@ -267,7 +267,6 @@ def forward(
         return objs
 
 
-# Copied from diffusers.models.unet_2d_condition.UNet2DConditionModel with UNet2DConditionModel->UNetFlatConditionModel, nn.Conv2d->LinearMultiDim, Block2D->BlockFlat
```

Comment:
@DN6 @patrickvonplaten 
do we intend to keep all the `#copied from` statements for deprecated pipelines? that means we still have to maintain them, no?